index.md

```diff
@@ -4,7 +4,7 @@ This is the Kubernetes Operator for Loki provided by the Red Hat OpenShift engin
 
 ### Installation of Storage Size Calculator on OpenShift
 
-* Storage size calculator is to be installed on OpenShift only.
-* The step-by-step guide on how to install [storage size calculator](https://github.com/ViaQ/loki-operator/blob/master/docs/storage_size_calculator.md) is available.
+* Storage size calculator works out of the box on OpenShift. For other distributions you should list the availability of required services, e.g. prometheus, scrape configuration for log-file-metric exporter, promsecret to access the custom prometheus URL.
```

Comment:
The user should not `list the availability`. I meant you as an author should provide a list o required services to install on none-openshift distribution in order to use the Storage-size-calculator